enterprise/settings.mdx

```diff
@@ -128,3 +128,23 @@ DEFAULT_ORG_ROLE=view_all
 ```
 
 Note you can run an [Enterprise self-check](/docs/enterprise/troubleshooting) to verify the connection and lookup LDAP information, but some configuration errors are only visible once logging in.
+
+## Sidekiq monitoring using Prometheus
+
+The metrics HTTP endpoint `/metrics` is available for the Prometheus scraping. Currently, it provides Sidekiq metrics using [sidekiq-prometheus-exporter gem](https://github.com/Strech/sidekiq-prometheus-exporter).
```

Comment:
From a grammatical perspective, I think the gem should be prefixed with "the". Separately, I think there should be no "the" prefixing Prometheus.

Maybe we can rewrite like this:

```suggestion
The metrics HTTP endpoint `/metrics` is available for Prometheus scraping. Currently, it provides worker queue metrics using the [sidekiq-prometheus-exporter gem](https://github.com/Strech/sidekiq-prometheus-exporter).
```